api/src/main/java/com/cloud/storage/VolumeApiService.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Map;
 
 import org.apache.cloudstack.api.command.user.volume.AttachVolumeCmd;
+import org.apache.cloudstack.api.command.user.volume.ChangeOfferingForVolume;
```

Comment:
Is this command class (to change offering)? If so, update class name to *Cmd notation, to keep same as other classes.